docs/guides/connect-aepp-to-wallet.md

```diff
@@ -101,4 +105,3 @@ Note:
 
 - The steps above are snippets taken from the full implementation of
   the [Simple æpp](https://github.com/aeternity/aepp-sdk-js/tree/master/examples/browser/aepp)
-
```

Comment:
@janmichek or @subhod-i (?) ensure that your ide supports `.editorconfig`: https://github.com/aeternity/aepp-sdk-js/blob/0c3cae6f83026b763e57255ec969b4812c17416c/.editorconfig#L8